pixiedust/display/display.py

```diff
@@ -228,8 +228,12 @@ def getPreferredOutputWidth(self):
 
     def getPreferredOutputHeight(self):
         ch = self.options.get("nostore_ch", None)
+        vh = self.options.get("nostore_vh", None) # viewport height
+        indialog = self.options.get('runInDialog', 'false') == 'true' # running in dialog
         if ch is not None:
             return float(ch)
+        elif vh is not None and not indialog:
+            return max(400, (float(vh) * 0.5)) # height set to 400px or 50% of viewport
```

Comment:
@vabarbosa did you mean min(400, (float(vh) * 0.5)) here? we want to limit the height of the chart isn't it?

Reply:
@DTAIEB i was thinking of having the chart height at half the browser viewport. if for some reason the user had the viewport real small (e.g., 500px) instead of using height of 250 we just default to 400. otherwise, the chart height maybe too small. i am not strongly tied this, so if half the viewport is not preferred, i can definitely change it.